index.html

```diff
@@ -4,78 +4,100 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-146635872-1"></script>
     <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+        window.dataLayer = window.dataLayer || [];
 
-      gtag('config', 'UA-146635872-1');
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+
+        gtag('js', new Date());
+
+        gtag('config', 'UA-146635872-1');
     </script>
     <title>NOAA Fisheries Integrated Toolbox</title>
-    <link type="text/css" rel="stylesheet" href="./static/simple.css" />
-    <meta name="google-site-verification" content="U6vzg6_XZ4aLFUQSw7xIysB7E3WtARBQnKU_gbCQWz4" />
```

Comment:
Not necessary blank might be lead to page/system error. It would be better not to insert that.